ecosystem/sep-0031.md

```diff
@@ -197,8 +217,6 @@ Each `fields` sub-object contains a key for each field name and an object with t
 * `choices`: (optional) list of possible values for the field.
 * `optional`: (optional) false if not specified.
  
-If `sender_sep12_type` or `receiver_sep12_type` are present in the response, the client must register the sender and receiver via [SEP-12](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md#customer-get) using the `type` argument provided. The receiving anchor uses this parameter to know what information needs to be collected on the customer.
```

Comment:
💡 I think you still needs this statement, but referencing the new fields? If this is sufficiently discussed elsewhere, no worries.

Reply:
I think we can omit it given that the definitions of the new `_required` fields have this language:
> If `true`, the sending anchor should use the appropriate value from `sep12_types` in a [SEP-12 GET /customer?type=](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md#customer-get) request to determine which fields should be collected.